ViteMaDose/ViewModels/County Selection/CountySelectionViewModel.swift

```diff
@@ -0,0 +1,46 @@
+//
```

Comment:
From what I see from Github, there is a space in the folder name. I guess it's better to avoid that ?